src/services/ethconversion/ethconversion.hooks.js

```diff
@@ -0,0 +1,52 @@
+import onlyInternal from '../../hooks/onlyInternal';
+import { getEthConversion } from './getEthConversionService';
+
+const getConversionRates = () => (context) => {
+  const { app, params } = context;
+
+  console.log(params);
+
+  // block internal calls
+  if (!params.provider && !params.internal) return context
+
+  console.log('getting conversion')
+
+  return getEthConversion(app, params.query.date)
+    .then((res) => {
+      context.result = res;
+      return context;
+    })
+}
+
+
+module.exports = {
+  before: {
+    all: [],
+    find: [],
+    get: [],
+    create: [onlyInternal()],
+    update: [onlyInternal()],
```

Comment:
do we have a need to update/patch/remove? If not, we should use the `disallow` hook.

we should also `disallow` `get` b/c it will return nothing